test/test_with_xpress.jl

```diff
@@ -896,6 +896,27 @@ end
     @test round(sum(cooling_electric_hybrid_expected .- cooling_elec_hybrid), digits=1) ≈ 0.0 atol=0.1
 end
 
+@testset "Boiler and Steam Turbine" begin
+    m1 = Model(optimizer_with_attributes(Xpress.Optimizer, "OUTPUTLOG" => 0))
+    m2 = Model(optimizer_with_attributes(Xpress.Optimizer, "OUTPUTLOG" => 0))
+    results = run_reopt([m1,m2], "./scenarios/boiler_steam_turbine.json")
+
+    # below copied from API test (TODO why so many unused values? only one test?)
```

Comment:
@Bill-Becker please look at this test copied from the API. There is only one value tested and it says `Expected value from spreadsheet`. Can we test more obvious expected values in addition to the net_electric_efficiency?